desktop/src/cli.rs

```diff
@@ -115,6 +115,10 @@ pub struct Opt {
     #[clap(long, value_parser)]
     pub spoof_url: Option<Url>,
 
+    /// Spoofs the URL of the page embedding the SWF.
+    #[clap(long, value_parser)]
+    pub spoof_page_url: Option<Url>,
```

Comment:
I don't think we need a separate option - we can extract the page url from `spoof_url`

Reply:
But maybe we want to spoof a domain for the SWF and the page? This option would be used for https://github.com/ruffle-rs/ruffle/pull/14015 as well, so I think we should add it.

Reply:
Do we know of any games that need different page and SWF URLs? If not, then I do think keeping it simple with one option is best

Reply:
I don't know any game off the top of my head that would need this. So let's remove it for now, adding it back is trivial.